### PR TITLE
[release/8.0-staging] Fix WinHttp StreamingTest backward compat version

### PR DESCRIPTION
**Pull request by liveans**
Disables BidirectionStreamingTests for WS2022

main PR #104722 which fixes the behavior, but in release branches we're disabling tests instead.

# Description

Recently, WS2022 got an update which backports AUTOMATIC_CHUNKING in WinHttp, and it affects our test in CI. Because, we were relying on it, now we're disabling tests because #104891 started to pop up with the backport in WS2022.
# Customer Impact

No Customer Impact, test-only change.
# Regression

No
# Testing

CI
# Risk

Test-only change, no risk.
# Package authoring signed off?

IMPORTANT: If this change touches code that ships in a NuGet package, please make certain that you have added any necessary [package authoring](../../docs/project/library-servicing.md) and gotten it explicitly reviewed.

### CODE REVIEW

**Comment by liveans** [comment]
This change is failing on `windows.amd64.server2022.open.svc` images but passing on `windows.amd64.server2022.open.rt` for both release/8.0-staging and release/6.0-staging, it's highly likely that open.svc is using older version on server2022.

**Review by rzikm — DISMISSED** [review]
LGTM, but please use the full servicing template for consistency.

**Comment by liveans** [comment]
> LGTM, but please use the full servicing template for consistency.

I will keep this on hold, until we figure out what's going on with failures in release branches.

File: src/libraries/System.Net.Http.WinHttpHandler/tests/FunctionalTests/BidirectionStreamingTest.cs
Comment (rzikm) [inline src/libraries/System.Net.Http.WinHttpHandler/tests/FunctionalTests/BidirectionStreamingTest.cs:25]: I assume this should be the same as in https://github.com/dotnet/runtime/pull/104838/files

```suggestion
        public static bool OsSupportsWinHttpBidirectionalStreaming => Environment.OSVersion.Version >= new Version(10, 0, 22357, 0) || PlatformDetection.IsWindowsServer2022;
```

**Review by rzikm — APPROVED** [review]
LGTM if Ci is green

**Comment by liveans** [comment]
Test only change, CI seems happy (windows passes), tell mode, adding servicing-approved

**Pull request merged**
